mono/mini/unwind.c

```diff
@@ -310,6 +310,7 @@ mono_print_unwind_info (guint8 *unwind_info, int unwind_info_len)
 		switch (op) {
 		case DW_CFA_advance_loc:
 			pos += *p & 0x3f;
+			printf ("CFA: [%x] advance loc\n",pos);
 			p ++;
```

Comment:
Its ok, but we don't print it because every instruction is prefixed by it.

Reply:
I added it so I could do a 1 for 1 comparison with what I was specifying and what was being processed. It took a read of the code to see why I wasn't seeing this so I just added it for completeness (least astonishment principle).